ourjapanlife/findadoc-web
# 17 husky linting

Resolves #17 

# What changed
- add husky hooks
- set up a script to run linter

# Testing instructions
1. run `yarn prepare`
2. make a new branch `git checkout -b whatever`
3. commit a change
4. push to origin
5. husky should run

1. run `yarn lint`

1. run `yarn lint:fix`
